.solcover.js

```diff
@@ -1,9 +1,13 @@
+const ganache = require('ganache-cli');
+const shared = require('./scripts/shared');
+
 module.exports = {
-    norpc: true,
-    testCommand: 'node --max-old-space-size=4096 ../node_modules/.bin/truffle test --network coverage',
-    compileCommand: 'node --max-old-space-size=4096 ../node_modules/.bin/truffle compile --network coverage',
-    skipFiles: [
-        'lifecycle/Migrations.sol',
-        'mocks'
-    ]
+  skipFiles: [
+    'lifecycle/Migrations.sol',
+    'mocks'
+  ],
+  port: 8545,
+  client: ganache,
+  onServerReady: shared.setupGSNRelayHub,
```

Comment:
Is there documentation for these options? I'm assuming SC will run `client` on `port` with the options provided, and then call `onServerReady`, but does this happen on a separate process?

Reply:
@nventuro 

Yes, there are [options docs here](https://github.com/sc-forks/solidity-coverage/tree/beta#config-options).

>  I'm assuming SC will run client on port with the options provided, and then call onServerReady, but does this happen on a separate process?

The ganache server is run the same process as truffle. In this PR, `onServerReady` executes a [helper method](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/1923/files#diff-f76afefa4e67ba58e941de1756c6b919R68) which uses child_process.exec to launch `oz-gsn` separately.